packages/forklift-console-plugin/src/modules/Plans/views/create/components/PlanCreateForm.tsx

```diff
@@ -74,64 +84,79 @@ export const PlanCreateForm: React.FC<PlanCreateFormProps> = ({
+                <ForkliftTrans>
+                  The project that your migration plan will be created in. Only projects with
+                  providers in them can be selected.
+                </ForkliftTrans>
+              </StackItem>
+
+              <StackItem>
+                <ForkliftTrans>
+                  Projects, also known as namespaces, separate resources within clusters.
+                </ForkliftTrans>
+              </StackItem>
+            </Stack>
           }
         />
 
-        <FormGroupWithHelpText fieldId="type">
-          <FiltersToolbarProviders
-            className="forklift--create-plan--filters-toolbar"
-            filterState={filterState}
-            filterDispatch={filterDispatch}
-          />
-          <ChipsToolbarProviders filterState={filterState} filterDispatch={filterDispatch} />
-
-          {filterState.selectedProviderUID ? (
-            <Flex>
-              <FlexItem className="forklift--create-provider-edit-card-selected">
-                <SelectableCard
-                  title={providerCardItems[filterState.selectedProviderUID]?.title}
-                  titleLogo={providerCardItems[filterState.selectedProviderUID]?.logo}
-                  onChange={() => onChange('')}
-                  isSelected
-                  isCompact
-                  content={
-                    <Tooltip
-                      content={
-                        <div>{t('Click to select a different provider from the list.')}</div>
-                      }
-                    >
-                      <HelperText>
-                        <HelperTextItem variant="indeterminate">
-                          {t('Click to unselect.')}
-                        </HelperTextItem>
-                      </HelperText>
-                    </Tooltip>
-                  }
-                />
-              </FlexItem>
-            </Flex>
-          ) : (
-            <SelectableGallery
-              selectedID={filterState.selectedProviderUID}
-              items={providerCardItems}
-              onChange={onChange}
+        {Object.values(providerCardItems).length ? (
+          <FormGroupWithHelpText fieldId="type">
+            <FiltersToolbarProviders
+              className="forklift--create-plan--filters-toolbar"
+              filterState={filterState}
+              filterDispatch={filterDispatch}
             />
-          )}
-        </FormGroupWithHelpText>
+            <ChipsToolbarProviders filterState={filterState} filterDispatch={filterDispatch} />
+
+            {filterState.selectedProviderUID ? (
+              <Flex>
+                <FlexItem className="forklift--create-provider-edit-card-selected">
+                  <SelectableCard
+                    title={providerCardItems[filterState.selectedProviderUID]?.title}
+                    titleLogo={providerCardItems[filterState.selectedProviderUID]?.logo}
+                    onChange={() => onProviderChange('')}
```

Comment:
the onChnage reset the provider?

Reply:
Yes, the way these components are setup is that once clicked, all other cards are hidden and this 1 SelectableCard is shown. In order to see the full "Gallery" of cards, you have to click this SelectableCard again which unsets the provider value.